js-exercises/least-common-multiple/leastCommonMultiple.js

```diff
@@ -0,0 +1,15 @@
+function leastCommonMultiple(...args) {
+  const largest = Math.max(args[0], args[1]);
+  const smallest = Math.min(args[0], args[1]);
+  let resultLCM = largest;
+
+  while (true) {
+    if (resultLCM % smallest === 0) {
```

Comment:
how would you check if the smallest number is 0 as it will throw exception

Reply:
Good catch. Ran some tests like `LCM(12,0)` it's running infinitely though not throwing any error.
Normally `LCM(12,0)` should output `0` .

Will fix the bug in code and mark this as resolved.